arm-sql/2014-04-01/swagger/importExport.json

```diff
@@ -0,0 +1,529 @@
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImportExportOperationStatusResponse"
+            }
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImportExportOperationStatusResponse"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ImportExportOperationStatusResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/importExportOperationResults/{operationId}": {
+      "get": {
+        "tags": [
+          "ImportExport"
+        ],
+        "operationId": "Databases_GetImportExportStatus",
+        "description": "The status of an import or export operation on an Azure SQL database given the operation ID.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "name": "databaseName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the Azure SQL database"
+          },
+          {
+            "name": "operationId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "description": "The operation ID associated with the operation."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImportExportOperationStatusResponse"
+            }
+          },
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ImportExportOperationStatusResponse"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ImportExportOperationStatusResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ImportExtensionProperties": {
+      "properties": {
+        "operationMode": {
+          "type": "string",
+          "description": "The type of Import/Export opertion being performed",
+          "enum": [
+            "Import",
+            "Export"
+          ],
+          "x-ms-enum": {
+            "name": "ImportExportOperationMode"
+          }
+        },
```

Comment:
OperationMode should be removed from the payload, from the API service is already aware of what is the operation mode, either user is calling import or he is calling export API.
Even in this API version the server can be changed not to depend on the operationMode property.